src/js/views/patients/schedule/schedule-list.scss

```diff
@@ -83,13 +83,20 @@
     cursor: default;
   }
 
+  &:hover {
+    .js-patient-sidebar-button {
+      color: $black-20;
+    }
+  }
+
   &:hover:not(.is-selected, .is-reduced) {
     background-color: color(light_blue, light);
   }
 }
 
 .schedule-list__action-list-cell {
   padding: 4px 8px;
+  width: calc(68% - 84px);
```

Comment:
For the text overflow ellipsis (`...`) styling to work, the `State, Action` column needed a width set to it.